src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -235,9 +236,18 @@ private void updateReport(final LocalDate dateToShow) {
 
          for (final Work w : onlyCurrentProjectWork) {
             final HBox workButtonBox = new HBox(5.0);
-            workButtonBox.getChildren().add(createCopyWorkButton(w));
-            workButtonBox.getChildren().add(createEditWorkButton(w));
-            workButtonBox.getChildren().add(createDeleteWorkButton(w));
+            if(w.getId()==model.activeWorkItem.get().getId()){
+               Label label = new Label("Active Work");
+               label.setTooltip(new Tooltip("The active work item cannot be edited as it is currently active. To edit it you need to switch to another work first."));
+               label.getTooltip().setFont(Font.font(15));
```

Comment:
not sure if we need this big size for the tooltip? had it a reason? (the other tooltips we have are not this big)

Reply:
Thought it might be hard to read if its size 12